e2e/libs/node/node.py

```diff
@@ -91,6 +98,63 @@ def wait_all_pods_evicted(self, node_name):
             if evicted:
                 break
 
-            time.sleep(RETRY_INTERVAL)
+            time.sleep(retry_interval)
 
         assert evicted, 'failed to evict pods'
+
+    def get_node_by_index(self, index, role="worker"):
```

Comment:
They are all node-related methods. Where do you suggest adding them? Or should we have a more specific python module for node operations like `node_ops.py`?

Reply:
I renamed the relevant node*.py files to node_instance*.py to distinguish them from the Node custom resource handling.

Reply:
`node.py` itself is the collection of node related operations. Like `volume. py` itself is the collection of volume related operations, and `workload.py` is the collection of node related operations, and so on.

But `get_node_by_index`, `list_node_names_by_role` etc. are just utility functions that getting information by calling some APIs and they don't involve any _operation_, so in my opinion, keeping them in a collection of utility functions is good enough. Having two files with similar naming (`node.py` and `node_instance.py`) seems doesn't make things better.

Reply:
These utility functions are for K8s nodes, but reboot, shut down operations are for physical nodes. We are confused because they are put together in the same folder/file now, but they are two completely different things and shouldn't have to be in the same folder. Probably creating two different folders: k8s_node and physical_node to separate them can make our lives easier. node.py and node_instance.py are still ambiguous to me.

Reply:
Sure, organized them by `host` and `node`, each having its own directory.